src/content/docs/concepts/drivers.md

```diff
@@ -4,4 +4,14 @@ title: Drivers
 
 A _driver_ is a [program](/concepts/programs/) whose sole responsibility is to re-expose another API as a Membrane program. You can see all the drivers maintained by Membrane in our [directory](https://github.com/membrane-io/directory).
 
-There's nothing fundamentally special about a driver. It's a program just like any other. That means you can create your own drivers for you or other Membrane users to use. If you do create a driver, feel free to submit a PR to our directory! We'd love to include it.
+There's nothing fundamentally special about a driver. It's a program just like any other. That means you can create your own drivers for you or other Membrane users to use.
+
+:::note
```

Comment:
Interesting. What is this syntax? I've never seen that before

Reply:
It's a markdown extension syntax [provided by Astro Starlight](https://starlight.astro.build/guides/authoring-content/#asides)